encoders/lf-att-ques-im-hist.lua

```diff
@@ -43,28 +43,34 @@ function encoderNet.model(params)
     local qh = nn.Tanh()(nn.Linear(2*params.rnnHiddenSize, params.rnnHiddenSize)(nn.JoinTable(1,1)({q3,h3})))
 
     -- image attention (inspired by SAN, Yang et al., CVPR16)
-    local img_feat_size = 512
+    local img_feat_size = {1, 14, 14, 512}
```

Comment:
I am not sure if it can be inferred at runtime.

Reply:
It creates the entire graph first, so can't be inferred. This looks good.